.github/workflows/continuous-integration.yml

```diff
@@ -1,21 +1,10 @@
-name: Tests
+name: CI
 
 on:
   push:
     branches:
       - main
   pull_request:
-  workflow_call:
```

Comment:
won't dropping this break end-to-end tests since they call this?

Reply:
seems like they are still fine: https://github.com/deepset-ai/deepset-cloud-e2e-tests/actions/runs/6767264115

Reply:
ah true - they call the `-dev` / `-prod` ones 👍🏻